City-of-Helsinki/julkaisut-helfi
# Drupal module updates, including Gutenberg and Facets security updates

# [UHF-10835](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10835)

## To test
* Checkout branch: `git checkout drupal-module-updates`
* Delete the old `dump.sql` file just in case (unless it's fresh).
* Run `make fresh`
* [x] Everything should install correctly
* [x] Check the available updates page, there should be no pending security updates: https://julkaisut-helfi.docker.so/fi/admin/reports/updates
* [x] Click around the site to check that things are working normally
* [x] Try to edit and create a few nodes (test that the Gutenberg module works after updates)
* [x] Reindex content with `drush sapi-c;drush sapi-rt;drush sapi-i`, this should complete without errors.
* [x] Test that the search functionality works: https://julkaisut-helfi.docker.so/fi/search
  * Note: You need to add a search term to get any results, just filtering by years returns nothing 
  * [x] There should be no twig markup printed on search results header.

## Updates:
```
- Upgrading composer/semver (3.4.2 => 3.4.3)
- Upgrading doctrine/annotations (1.14.3 => 1.14.4)
- Upgrading drupal/admin_toolbar (3.4.2 => 3.5.0)
- Upgrading drupal/core (10.3.1 => 10.3.6)
- Upgrading drupal/core-composer-scaffold (10.3.1 => 10.3.6)
- Upgrading drupal/core-recommended (10.3.1 => 10.3.6)
- Upgrading drupal/custom_book_block (1.0.1 => 1.1.0)
- Upgrading drupal/easy_breadcrumb (2.0.7 => 2.0.8)
- Upgrading drupal/entity (1.4.0 => 1.5.0)
- Upgrading drupal/eu_cookie_compliance (1.24.0 => 1.25.0)
- Upgrading drupal/facets (2.0.7 => 2.0.9)
- Upgrading drupal/gtm (1.8.0 => 1.9.0)
- Upgrading drupal/gutenberg (2.9.0 => 2.14.0)
- Upgrading drupal/helfi_api_base (2.7.6 => 2.7.9)
- Upgrading drupal/helfi_azure_fs (2.0.2 => 2.0.7)
- Upgrading drupal/media_library_edit (3.0.3 => 3.0.4)
- Upgrading drupal/metatag (2.0.0 => 2.0.2)
- Upgrading drupal/monolog (3.0.3 => 3.0.4)
- Upgrading drupal/pathauto (1.12.0 => 1.13.0)
- Upgrading drupal/raven (5.0.16 => 5.0.19)
- Upgrading drupal/redirect (1.9.0 => 1.10.0)
- Upgrading drupal/redis (1.7.0 => 1.8.0)
- Upgrading drupal/search_api_solr (4.3.4 => 4.3.5)
- Upgrading drupal/simple_sitemap (4.1.9 => 4.2.1)
- Upgrading drupal/smart_trim (2.1.1 => 2.2.0)
- Upgrading drupal/smtp (1.3.0 => 1.4.0)
- Upgrading drupal/stage_file_proxy (3.1.0 => 3.1.3)
- Upgrading drupal/token (1.14.0 => 1.15.0)
- Upgrading drupal/tvi (2.0.0-alpha2 => 2.0.0)
- Upgrading drupal/views_infinite_scroll (2.0.2 => 2.0.3)
- Upgrading drush/drush (12.5.2 => 12.5.3)
- Upgrading league/mime-type-detection (1.15.0 => 1.16.0)
- Upgrading maennchen/zipstream-php (3.1.0 => 3.1.1)
- Upgrading nikic/php-parser (v5.1.0 => v5.3.1)
- Upgrading phootwork/collection (v3.2.2 => v3.2.3)
- Upgrading phootwork/lang (v3.2.2 => v3.2.3)
- Upgrading phpmailer/phpmailer (v6.9.1 => v6.9.2)
- Upgrading phrity/util-errorhandler (1.1.0 => 1.1.1)
- Upgrading psr/log (3.0.0 => 3.0.2)
- Upgrading sentry/sentry (4.8.1 => 4.9.0)
- Upgrading symfony/console (v6.4.10 => v6.4.12)
- Upgrading symfony/dependency-injection (v6.4.10 => v6.4.12)
- Upgrading symfony/filesystem (v6.4.9 => v6.4.12)
- Upgrading symfony/finder (v6.4.10 => v6.4.11)
- Upgrading symfony/http-foundation (v6.4.10 => v6.4.12)
- Upgrading symfony/http-kernel (v6.4.10 => v6.4.12)
- Upgrading symfony/mailer (v6.4.9 => v6.4.12)
- Upgrading symfony/mime (v6.4.9 => v6.4.12)
- Upgrading symfony/polyfill-php72 (v1.30.0 => v1.31.0)
- Upgrading symfony/polyfill-php80 (v1.30.0 => v1.31.0)
- Upgrading symfony/polyfill-php81 (v1.30.0 => v1.31.0)
- Upgrading symfony/process (v6.4.8 => v6.4.12)
- Upgrading symfony/psr-http-message-bridge (v6.4.10 => v6.4.11)
- Upgrading symfony/routing (v6.4.10 => v6.4.12)
- Upgrading symfony/serializer (v6.4.10 => v6.4.12)
- Upgrading symfony/string (v6.4.10 => v6.4.12)
- Upgrading symfony/validator (v6.4.10 => v6.4.12)
- Upgrading symfony/var-dumper (v6.4.10 => v6.4.11)
- Upgrading symfony/yaml (v6.4.8 => v6.4.12)
- Upgrading twig/twig (v3.10.3 => v3.14.0)
```

[UHF-10835]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
